### PR TITLE
remove unneeded constraints

### PR DESCRIPTION
**Pull request by ashultz0**
constraints file had a couple of entries "matching platform" which platform must have removed a long time ago. Cleaning them up didn't actually change anything in the requirements.txt files but cleaning is worth doing anyway.

### CODE REVIEW

File: requirements/common_constraints.txt [hunk 1]
```diff
@@ -20,6 +20,3 @@ Django<4.0
 elasticsearch<7.14.0
 
 setuptools<60
-
-# redis 4 client doesn't play nicely with redis 3 server
-redis<4
```
Comment (alangsto) [inline requirements/common_constraints.txt:25]: Do we know if we're not using a redis 3 server anymore?

Comment (ashultz0) [reply requirements/common_constraints.txt:25]: common_constraints is actually copied out of https://github.com/openedx/edx-lint/blob/master/edx_lint/files/common_constraints.txt by make upgrade, it looks like the redis constraint was recently removed

Comment (alangsto) [reply requirements/common_constraints.txt:25]: Nice

**Review by alangsto — APPROVED** [review]
👍

**Pull request merged**